pkg/util/github/github.go

```diff
@@ -190,7 +190,9 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	binaryPath := ""
 	switch {
 	case strings.HasSuffix(assetTmpFile, TAR):
-		tarFile := assetTmpFile[:len(assetTmpFile)-3]
+		tokens := strings.Split(assetFilename, fmt.Sprintf("%s-", version))
```

Comment:
Instead of hacking directory names, we can work around this by just unpacking the tarball and reading the actual created directory from the file system. Much more resilient. This will work regardless of what Origin will do, since it won't be based on assumptions.